package.json

```diff
@@ -53,7 +53,38 @@
           "description": "Run the linter on save (onSave) or on type (onType)."
         }
       }
-    }
+    },
+    "languages": [
+      {
+        "id": "json",
+        "filenames": [
+          ".pug-lintrc",
+          ".jade-lintrc"
+        ]
+      }
+    ],
+    "jsonValidation": [
+      {
```

Comment:
Please, use one glob-pattern for all files. For example, `.{pug,jade}-lint{rc,}{.json,}` or `*{pug,jade}-lint{rc,}{.json,}`. But test it, because i'm not sure that VS Code support glob with brackets 😄